src/app/warehouse/material.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#include <cstdint>
+#include <string>
+
+enum class Material : std::uint8_t
+{
+    DentalCement = 0,
+    Amalgam = 1,
+    DentalComposite = 2,
+    Size = 255
+};
+
+inline std::string convertMaterial(const Material &material)
```

Comment:
What if we pass null as an argument?

Reply:
C++ strong typing ensures that we cannot pass null as an argument.